MAUI/ProgressBar/migration.md

```diff
@@ -0,0 +1,725 @@
+                                   ProgressFill="Red"
+                                   TrackFill="Violet"
+                                   Progress="75"
+                                   IsIndeterminate="False"
+                                   Minimum="10"
+                                   Maximum="90"
+                                   AnimationDuration="1500"
+                                   IndeterminateAnimationDuration="2000">
+    <progressBar:SfCircularProgressBar.GradientStops>
+        <progressBar:ProgressGradientStop Color="#00bdaf" Value="0"/>
+        <progressBar:ProgressGradientStop Color="#2f7ecc" Value="50"/>
+    </progressBar:SfCircularProgressBar.GradientStops>
+
+</progressBar:SfCircularProgressBar>
+...
+
+</ContentPage>
+ 
+{% endhighlight %}
+
+{% highlight C# %}
+
+using Syncfusion.Maui.ProgressBar;
+...
+//Snippet for the linear progress bar
+
+SfLinearProgressBar linearProgressBar = new SfLinearProgressBar();
+linearProgressBar.AnimationEasing = Easing.CubicOut;
+linearProgressBar.Progress = 75;
+linearProgressBar.Minimum = 10;
+linearProgressBar.Maximum = 90;
+linearProgressBar.SegmentGapWidth = 5;
+linearProgressBar.SegmentCount = 4;
+linearProgressBar.IndeterminateAnimationEasing = Easing.BounceIn;
+linearProgressBar.IndeterminateIndicatorWidthFactor = 0.7;
+linearProgressBar.AnimationDuration = 1500;
+linearProgressBar.IndeterminateAnimationDuration = 2000;
+linearProgressBar.IsIndeterminate = false;
+linearProgressBar.ProgressFill = Colors.Red;
+linearProgressBar.TrackFill = Colors.Violet;
+linearProgressBar.GradientStops.Add(new ProgressGradientStop { Color = Color.FromArgb("00bdaf"), Value = 0 });
+linearProgressBar.GradientStops.Add(new ProgressGradientStop { Color = Color.FromArgb("2f7ecc"), Value = 50 });
+this.Content = linearProgressBar;
+
+//Snippet for the circular progress bar
+
+SfCircularProgressBar circularProgressBar = new SfCircularProgressBar();
+circularProgressBar.AnimationEasing = Easing.CubicOut;
+circularProgressBar.Progress = 75;
+circularProgressBar.Minimum = 10;
+circularProgressBar.Maximum = 90;
+circularProgressBar.SegmentGapWidth = 5;
+circularProgressBar.SegmentCount = 4;
+circularProgressBar.IndeterminateAnimationEasing = Easing.BounceIn;
+circularProgressBar.IndeterminateIndicatorWidthFactor = 0.7;
+circularProgressBar.AnimationDuration = 1500;
+circularProgressBar.IndeterminateAnimationDuration = 2000;
+circularProgressBar.IsIndeterminate = false;
+circularProgressBar.ProgressFill = Colors.Red;
+circularProgressBar.TrackFill = Colors.Violet;
+circularProgressBar.GradientStops.Add(new ProgressGradientStop { Color = Color.FromArgb("00bdaf"), Value = 0 });
+circularProgressBar.GradientStops.Add(new ProgressGradientStop { Color = Color.FromArgb("2f7ecc"), Value = 50 });
+this.Content = circularProgressBar;
+
+...
+
+{% endhighlight %}
+
+{% endtabs %}
+</td>
+</tr>
+</table>
+
+
+## SfLinearProgressBar
+
+<table> 
```

Comment:
Before adding the code snippet, provide the below sentence

The following code example explains how to initialize the Xamarin SfLinearProgressBar and the .NET MAUI SfCircularProgressBar along with its properties.